dashboard/dashapi/dashapi.go

```diff
@@ -308,6 +311,7 @@ type BugReport struct {
 	CreditEmail       string // email for the Reported-by tag
 	Maintainers       []string
 	CC                []string // additional CC emails
+	Recipients        Recipients
```

Comment:
Does this replace Maintainers and Cc?  If so, can you leave a comment noting that Maintainers and Cc are deprecated, similar to the other cases?